src/app/src/main/kotlin/org/imperial/mrc/mint/Extensions.kt

```diff
@@ -35,6 +35,7 @@ fun Response.asResponseEntity(): ResponseEntity<String> {
 
 }
 
+@Suppress("MagicNumber")
```

Comment:
This seemed like a pretty benign set of of magic numbers..

Reply:
I actually changed this over in HINT to avoid the magic numbers, since you can get them all from the `HttpStatus` class:

```
fun httpStatusFromCode(code: Int): HttpStatus {
    val status = HttpStatus.resolve(code) ?: return HttpStatus.INTERNAL_SERVER_ERROR
    return if (status <= HttpStatus.NOT_FOUND) {
        status
    } else {
        HttpStatus.INTERNAL_SERVER_ERROR
    }
}
```